CDC-Data-Reconciliation-Backend/server.py

```diff
@@ -243,6 +243,20 @@ async def get_report_cases(report_id: int):
         raise HTTPException(status_code=404, detail="Report not found")
     return report
 
+@app.get("/report_statistics/{report_id}")
+async def get_report_statistics(report_id: int):
+    try:
+        cur = app.liteConn.cursor()
+        cur.execute(
+            "SELECT * FROM Statistics WHERE ReportID = ?", (report_id,))
+        results = cur.fetchall()
+        if not results:
+            raise HTTPException(status_code = 404, detail = "Report Not Found")
+        return [dict(zip([column[0] for column in cur.description], row)) for row in results]
+    
+    except sqlite3.Error as e:
+        print(f"Database error: {e}")
+        raise HTTPException(status_code = 500, detail = "Internal Server Error")
```

Comment:
The code looks to do what it's intended to do, no qualms about this. You should be good to go!